tests/scale/threaded_utils/virt.py

```
@@ -0,0 +1,240 @@
+        sampler = TimeoutSampler(
+            wait_timeout=wait_timeout,
+            sleep=1,
+            func=_get_virt_launcher_instance,
+        )
+        try:
+            sample = None
+            for sample in sampler:
+                if sample and sample.spec.nodeName:
+                    return
+        except TimeoutExpiredError:
+            LOGGER.error(f"VM: {_vm.name} Status: {_vm.instance.status} virt-launcher: {sample}")
```

Ruff (0.14.8)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)